### PR TITLE
🏗 Remove experiment screen from amp-story 1.0

### LINKED ISSUE #11475: Cleanup amp-story-v1 experiment
Tracking issue #14357

Experiment name `amp-story-v1`

### PR DESCRIPTION
**Pull request by newmuis**
Fixes #11475

### CODE REVIEW

**Comment by gmajoulet** [comment]
Could you please also remove the unit tests from `1.0/test/test-amp-story.js`?

**Comment by newmuis** [comment]
@gmajoulet Done!

File: extensions/amp-story/1.0/amp-story.js [hunk 1]
```diff
@@ -719,102 +710,6 @@ export class AmpStory extends AMP.BaseElement {
...
-    errorIconEl.classList.add('i-amphtml-story-experiment-icon-error');
-
-    const errorMsgEl = this.win.document.createElement('span');
-    errorMsgEl.textContent = this.localizationService_.getLocalizedString(
-        LocalizedStringId.AMP_STORY_WARNING_EXPERIMENT_DISABLED_TEXT);
-
-    const experimentsLinkEl = this.win.document.createElement('button');
-    experimentsLinkEl.textContent = this.localizationService_
-        .getLocalizedString(
-            LocalizedStringId.AMP_STORY_EXPERIMENT_ENABLE_BUTTON_LABEL);
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story.js]: Sorry for the back and forth, but could you please deprecate the `LocalizedStringId`?

Comment (newmuis) [reply extensions/amp-story/1.0/amp-story.js]: Thanks for catching this!  No need to apologize 😃

**Pull request merged**